tests/osaka/eip7692_eof_v1/eip4200_relative_jumps/test_rjump.py

```diff
@@ -134,26 +118,22 @@ def test_rjump_truncated_rjump_2(
 ):
     """EOF1I4200_0002 (Invalid) EOF code containing truncated RJUMP"""
     eof_test(
-        data=Container(
-            sections=[Section.Code(code=Op.RJUMP + Op.STOP)],
-        ),
+        data=Container.Code(Op.RJUMP + Op.STOP),
```

Comment:
What is good replacement? `Op.RJUMP + "00"`?

Reply:
I did cause confusion for me at first glance. What about allowing arbitrary data portions with when argument in the square brackets is already a bytes-like object. E.g.
- `Op.RJUMP[0]` -> `0xef0000`
- `Op.RJUMP[b"\0"]` -> `0xef00`
- `Op.RJUMP[b""]` -> `0xef`

Not for this PR anyway because I'm not sure how many tests such a change would break.

Reply:
Fine to me.

Allowing verbatim hex in the code is also good to me. E.g. we sometimes need to use undefined opcodes.

Reply:
verbatim hex is great.

Reply:
This works currently: `Op.RJUMP + Bytecode(b"\x00")`. Should I change it?

Reply:
> Op.RJUMP + Bytecode(b"\x00")

Yes I think this is our best choice at the moment.

Separately, I'm going to prepare the PR to allow verbatim bytes, but it's going to take me a bit to make sure it does not break existing tests.

Reply:
Ok. I will update these tests with `Bytecode(b"\x00")`.

I can also try the separate PR mentioned. Is this about overloading `+` operator?

Reply:
Ah, actually the `Bytecode(b"\x00")` is broken because Bytecode tries to read some metadata of the argument.

What works is: `bytes(Op.RJUMP) + b"\x00"`
?

Reply:
Let's leave it as is, and I will prepare a PR for the verbatim data portion in bytes.